menstruation/handlers.py

```diff
@@ -13,36 +12,28 @@
 from telegram.ext import CallbackContext
 
 import menstruation.client as client
-from menstruation.config import MenstruationConfig
+from menstruation import config
```

Comment:
how is this better than `menstruation.config`? i think it was nicely parallel to `menstruation.{client,query}`

Reply:
I wanted config to be initailized immediately on the first import based on the environment variables and then to be able to use and modify golobal from any other module. Thus config becomes independent.

Reply:
:ok: :cool: